res/database/Default/user/stukov_ads.xml

```diff
@@ -377,8 +378,8 @@
 				<en>The brand you trust. Now as a unique DIY component set.</en>
 				<pl>Marka, której ufasz. Teraz jako unikalny zestaw komponentów Zrób To Sam.</pl>
 			</description>
-			<conditions min_audience="1.04" min_image="0" target_group="0" pro_pressure_groups="0" contra_pressure_groups="0" />
-			<data quality="34" repetitions="4" duration="5" profit="900" penalty="1700" infomercial_profit="103" fix_infomercial_profit="1" />
+			<conditions min_audience="4.1671" min_image="0" target_group="0" pro_pressure_groups="0" contra_pressure_groups="0" />
```

Comment:
I see no reason to change the parameters and thought it was done by "mistake".

Was intending to use the replaced parameter settings for some new ad, but seems didn't so far. Applying now to 'Bolton Dynamos'.

Reply:
The intention of changing parameters could be that over time the company's reputation/success changes (starting as laughing stock becoming a major player or vice versa).
I still think that changing the values for known ad considerably might be confusing for the players.

Reply:
Yeah I favor the parameters for the different era versions of an ad to stay the same, at least roughly, so that each era roughly plays the same. There can be exceptions, but I think these should be "explained" with humor then.